File: SocialSignIn_java/app/src/main/java/com/androprogrammer/socialsignin/listeners/SocialConnectListener.java
package com.androprogrammer.socialsignin.listeners;

import com.androprogrammer.socialsignin.model.UserLoginDetails;

/**
 * Created by Wasim on 06-Dec-15.
 */
public interface SocialConnectListener {

	public void onUserConnected(int requestIdentifier,UserLoginDetails userData);
	public void onConnectionError(int requestIdentifier,String message);
	public void onCancelled(int requestIdentifier,String message);
}

File: SocialSignIn_kotlin/app/src/main/kotlin/com/androprogrammer/socialsignin/listeners/SocialConnectListener.kt
package com.androprogrammer.socialsignin.listeners

import com.androprogrammer.socialsignin.model.UserLoginDetails

/**
 * Created by Wasim on 06-Dec-15.
 */
interface SocialConnectListener {

    fun onUserConnected(requestIdentifier: Int, userData: UserLoginDetails)
    fun onConnectionError(requestIdentifier: Int, message: String)
    fun onCancelled(requestIdentifier: Int, message: String)
}
